### PR TITLE
[Bug] Fix Open Street Maps Ajax

### PR DESCRIPTION
**Pull request by kingjia90**
Cherry pick from https://github.com/pimcore/pimcore/pull/15689
Bumped `admin ui bundle` minimum to `1.0.6` because `pimcore.helpers.sendRequest()` would be introduced in https://github.com/pimcore/admin-ui-classic-bundle/pull/210

### CODE REVIEW

**Comment by blankse** [comment]
Target branch should bei `1.0`

File: public/js/settings/conditions.js [hunk 1]
```diff
@@ -258,18 +258,18 @@ pimcore.bundle.personalization.settings.conditions = (function () {
...
-                            var address = searchfield.getValue();
-                            Ext.Ajax.request({
-                                url: pimcore.bundle.personalization.settings.conditions.getSearchUrl(address),
-                                method: "GET",
-                                success: function (response, opts) {
-                                    var data = Ext.decode(response.responseText);
+                            const address = searchfield.getValue();
+                            pimcore.helpers.sendRequest(
+                                "GET",
+                                pimcore.settings.targeting.conditions.getSearchUrl(address),
```
Comment (aryaantony92) [inline public/js/settings/conditions.js]:
```suggestion
                                pimcore.bundle.personalization.settings.conditions.getSearchUrl(address),
```

Comment (kingjia90) [reply public/js/settings/conditions.js]: thank you, nice catch! 
I didn't noticed it when copy/paste from 10.6 🤦🏻

Comment (aryaantony92) [reply public/js/settings/conditions.js]: No problem :)

**Review by aryaantony92 — APPROVED** [review]

**Pull request merged**